blocks/commerce-company-create/README.md

```diff
@@ -0,0 +1,80 @@
+# Commerce Company Create Block
+
+## Overview
+
+The Commerce Company Create block provides a company registration form for B2B customers. It integrates with the Adobe Commerce Company Management dropin to handle company registration workflows, including frontend configuration checks, authentication validation, and form submission.
+
+## Integration
+
+<!-- ### Block Configuration
+
+No block configuration is read via `readBlockConfig()`. -->
+
+<!-- ### URL Parameters
+
+No URL parameters affect this block's behavior. -->
+
+<!-- ### Local Storage
+
+No localStorage keys are used by this block. -->
+
+### Events
+
+#### Event Listeners
+
+- `events.on('b2b/config', callback)` - Listens for B2B configuration status updates from scripts/commerce-b2b.js
+- Authentication state changes are handled through the commerce framework
```

Comment:
If this is not done in the block, it can be removed and the section commented out.